### PR TITLE
Added limitations to docs: Python app support and Spark Shell.

### PR DESCRIPTION
**Pull request by susanxhuynh**
Please review. Thanks. @mgummelt @sascala cc/ @triclambert @keithchambers

### CODE REVIEW

File: docs/user-docs.md [hunk 1]
```diff
@@ -676,11 +676,14 @@ output:
 *   DC/OS Spark only supports submitting jars and Python scripts. It
 does not support R.
 
+*   Mesosphere does not provide support for Python app development.
```
Comment (mgummelt) [inline docs/user-docs.md]: We don't provide support for Scala app development either, right?

Comment (susanxhuynh) [inline docs/user-docs.md]: How about: "Mesosphere does not provide support for Spark app development."

Comment (mgummelt) [inline docs/user-docs.md]: :+1:

**Comment by susanxhuynh** [comment]
I just realized my branch was out of date and I'll have to merge my changes into the new file structure (where user-docs.md is split into multiple files).

**Comment by susanxhuynh** [comment]
I created a new PR rather than deal with a complicated merge.

**Pull request closed without merging**